frontend/src/Components/UserProfileKarma.tsx

```diff
@@ -60,23 +70,73 @@ export const UserProfileKarma = (props: UserProfileKarmaProps) => {
     const postSubsites = karmaResult && Object.keys(karmaResult.postRatingBySubsite);
     const commentSubsites = karmaResult && Object.keys(karmaResult.commentRatingBySubsite);
 
+    const restrictions = restrictionsResult && [
+        !restrictionsResult.canInvite &&
+        <><span className={'i i-can-not-invite'}></span>
+            <div>
+                Нет права приглашать других людей на орбитар.
+                {restrictionsResult.effectiveKarma >= 0 && <>
+                   <p>Для новичков это нормально, они получают это право после того, как начитают писать хорошие посты и комменты.</p>
+                </>}
+            </div>
+        </>,
+
+        restrictionsResult.commentSlowModeWaitSec > 0 &&
+        <><span className={'i i-slow'}></span>Право комментировать ограничено одним комментарием в {formatTimeSec(restrictionsResult.commentSlowModeWaitSec)}.</>,
```

Comment:
this probably won't happen in real life but `formatTimeSec` needs to be fixed to use another case:

<img width="910" alt="image" src="https://user-images.githubusercontent.com/57953386/184730284-9de60048-f92f-4e19-9a69-c294f8d72463.png">

Reply:
How to fix: https://github.com/moment/moment/blob/develop/locale/ru.js#L24

Reply:
<img width="908" alt="image" src="https://user-images.githubusercontent.com/2865203/184779526-84111d8c-525b-4355-8e8e-0b1f167dafa9.png">